generic/tests/trans_hugepage_relocated.py

```diff
@@ -73,21 +73,20 @@ def nr_hugepage_check(sleep_time, wait_time):
         mem_free_filter = r"MemFree:\s+(.\d+)\s+(\w+)"
         guest_mem_free, guest_unit = re.findall(mem_free_filter, o)[0]
         if re.findall("[kK]", guest_unit):
-            guest_mem_free = str(int(guest_mem_free) / 1024)
+            guest_mem_free = int(guest_mem_free) / 1024
```

Comment:
python 3 you will get a float by doing this
>>> 2000/1024
1.953125

Reply:
use `//`?

Reply:
good points :)